consensus/src/marshal/mod.rs

```diff
@@ -405,7 +405,7 @@ mod tests {
                     // Always finalize 1) the last block in each epoch 2) the last block in the chain.
                     // Otherwise, finalize randomly.
                     if height == NUM_BLOCKS
-                        || utils::is_last_block_in_epoch(BLOCKS_PER_EPOCH, epoch).is_some()
+                        || utils::is_last_block_in_epoch(BLOCKS_PER_EPOCH, height).is_some()
```

Comment:
This happened in a test and it's meaningless, but it's a bug that the new types caught!